en/middleware/micrortps.md

```diff
@@ -73,25 +82,24 @@ The *Agent* must be separately/manually compiled for the target computer.
 > **Tip** The bridge code can also be [manually generated](micrortps_manual_code_generation.md). 
   Most users will not need to do so, but the linked topic provides a more detailed overview of the build process and can be useful for troubleshooting.
 
-### ROS2/ROS applications - `px4_ros_com`
+### ROS2/ROS applications {#px4_ros_com}
 
-The [px4_ros_com](https://github.com/PX4/px4_ros_com) package allows the developer to generate all the required ROS2/ROS message headers and sources, and the agent application that will bridge the client with the RTPS middleware on the ROS2 side.
+The [px4_ros_com](https://github.com/PX4/px4_ros_com) package, when built, generates everything needed to access PX4 uORB messages from a ROS2 node (for ROS1 you also need [ros1_bridge](https://github.com/ros2/ros1_bridge)).
+This includes all the required components of the *PX4 RTPS bridge, including the IDL files (required by the `micrortps_agent`), the `micrortps_agent` itself and the sources and headers of the ROS messages.
 
 The package has two separate branches:
 - a `master` branch, used with ROS2. It contains code to generate all the required ROS2 messages and IDL files to bridge PX4 with ROS2 nodes.
-- `ros1` is used with ROS(1). It contains code to generate the ROS message headers and source files, which can be used *with* the `ros1_bridge` to share data between PX4 and ROS.
+- a `ros1` branch, used with ROS(1). It contains code to generate the ROS message headers and source files, which can be used *with* the `ros1_bridge` to share data between PX4 and ROS.
```

Comment:
After this might be good to add a note at the stating what about building the client for ROS?

Reply:
The client side is built the same way for a standart RTPS app or for a ROS2 app.

Reply:
Also, maybe it makes sense to have a sinlr way of identifying ROS 1. There's ROS1 in some places, ROSv1 on others. Maybe initially we should state that ROS v1 is from now on, called only ROS (because that's how it is still called, even now that you have ROS2).

Reply:
It seems to me that you really want to just call it ROS, so that is what I have done (what I would like to have done is just always referred to it as ROS1).